<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CambiarElTamano</class>
 <widget class="QMainWindow" name="CambiarElTamano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CambiarElTamano</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>239</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <widget class="QMenu" name="menuVista">
    <property name="title">
     <string>Vista</string>
    </property>
    <addaction name="actionZoom_Acercar"/>
    <addaction name="actionZoom_Alejar"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVista"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_Acercar"/>
   <addaction name="actionZoom_Alejar"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="toolTip">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionZoom_Acercar">
   <property name="text">
    <string>Zoom Acercar</string>
   </property>
  </action>
  <action name="actionZoom_Alejar">
   <property name="text">
    <string>Zoom Alejar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
